gems/sorbet/lib/serialize.rb

```diff
@@ -223,9 +223,9 @@ def constant(const, value)
     if KEYWORDS.include?(const.to_sym)
       return "# Illegal constant name: #{const}"
     end
-    if defined?(T::Types) && Sorbet::Private::RealStdlib.real_is_a?(value, T::Types::TypeMember)
+    if defined?(T::Types::TypeMember) && Sorbet::Private::RealStdlib.real_is_a?(value, T::Types::TypeMember)
       value.variance == :invariant ? "  #{const} = type_member" : "  #{const} = type_member(#{value.variance.inspect})"
-    elsif defined?(T::Types) && Sorbet::Private::RealStdlib.real_is_a?(value, T::Types::TypeTemplate)
+    elsif defined?(T::Types::TypeTemplate) && Sorbet::Private::RealStdlib.real_is_a?(value, T::Types::TypeTemplate)
```

Comment:
Why does this change show up?

Reply:
I was getting test failures without it, as the TypeMember and TypeTemplate constants were used before being defined. I have no idea how this was happening, so it would be good to try the tests again and see if it's still necessary. This was arising from the snapshot tests, so they're easy to run locally.